receiver/podmanreceiver/config.go

```diff
@@ -36,6 +36,9 @@ type Config struct {
 	APIVersion    string `mapstructure:"api_version"`
 	SSHKey        string `mapstructure:"ssh_key"`
 	SSHPassphrase string `mapstructure:"ssh_passphrase"`
+
+	// A list of filters whose matching images are to be excluded. Supports literals, globs, and regex.
+	ExcludedImages []string `mapstructure:"excluded_images"`
```

Comment:
```suggestion

	Exclude ExcludeConfig `mapstructure:"exclude"`
}

type ExcludeConfig struct {
	Container ExcludeContainerConfig `mapstructure:"container"`	
}

type ExcludeContainerConfig struct {
	// A list of filters whose matching images are to be excluded. Supports literals, globs, and regex.
	Images []string `mapstructure:"images"`	
```


What do you think about the following config structure?

We can exclude containers by labels in the future for example.

Reply:
Definitely, that looks much more flexible. Just a small question, the receiver configuration would look like:

```yaml
exclude:
  container:
    images:
      - *nginx*
```
Why do we need the `container` mapstructure? Could we just use the `ExcludeContainerConfig`? Is it because the ability to extend the exclude option to other types apart from containers? To exclude some kind of metrics?

Reply:
> Is it because the ability to extend the exclude option to other types apart from containers? 

Yep, I think about this. Maybe podmanreceiver will collect volume metrics, for example.

But it's up to you.

Reply:
Makes sense to me, added in : https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14397/commits/01b7f1ccf1eb1bc4dbb8533707fdd1b34bc2f881

Thanks for the proposal!

Reply:
Same Q here. Why do we need `container`? What is a possible use case?